src/Http/RequestFactory.php

```diff
@@ -218,4 +231,38 @@ public function createHttpRequest()
 		return new Request($url, NULL, $post, $files, $cookies, $headers, $method, $remoteAddr, $remoteHost, $rawBodyCallback);
 	}
 
+	/**
+	 * @param bool $usingTrustedProxy
```

Comment:
There should be two spaces between `@param` and `bool` (to align with `return`). The name of the parameter (`$usingTrustedProxy`) should be removed.